neonvm/controllers/virtualmachine_controller.go

```diff
@@ -379,8 +411,9 @@ func (r *VirtualMachineReconciler) doReconcile(ctx context.Context, virtualmachi
 			log.Info("Runner Pod was created", "Pod.Namespace", pod.Namespace, "Pod.Name", pod.Name)
 
 			r.Recorder.Event(virtualmachine, "Normal", "Created",
-				fmt.Sprintf("VirtualMachine %s created, pod %s",
-					virtualmachine.Name, pod.Name))
+				fmt.Sprintf("VirtualMachine %s created, Pod %s, SSH Secret %s",
```

Comment:
This is not your change, but `virtualmachine.Name` is redundant, it's already included in an event.

Reply:
Good point. We should fix this in the entire controller codebase. I'll make an issue for it.

Reply:
Opened https://github.com/neondatabase/autoscaling/issues/733